invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentStats/DocumentStats.js

```diff
@@ -28,8 +28,15 @@ export default class DocumentStats extends Component {
   }
 
   renderStats() {
-    const { isLoading, error, data } = this.props;
-    let renewalCount = sumBy(data.hits, loan => loan.metadata.extension_count);
+    const { isLoading, error, data, document } = this.props;
+    const pastLoans = data.total;
+    const renewalCount = sumBy(
+      data.hits,
+      loan => loan.metadata.extension_count
+    );
+    let avgLoans = pastLoans / document.metadata.items.total;
```

Comment:
It took me a while to understand the need of `isFinite` and `Math.round` :)

What about:
```
const pastLoans = data.total || 0;
const avgLoans = document.metadata.items.total ? (pastLoans / document.metadata.items.total).toFixed(2)  : "-"
```

Reply:
Maybe even one decimal is enough for this value?